src/components/pages/HomePage.jsx

```diff
@@ -1,31 +1,28 @@
-import React from "react";
-import { useNavigate } from "react-router-dom";
-import mainImage from "../../assets/imgMainPage.jpg";
-import GreenBlobImage from "../../assets/blob_green.png";
-import YellowBlobImage from "../../assets/blob_yellow.png";
-import PurpleBlobImage from "../../assets/blob_dusty_purple.png";
-import items from "../../data/data";
-import ArrowUp from "../../assets/icons/icons-arrow-up.png";
-import ArrowSearch from "../../assets/icons/icon-search.svg";
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import mainImage from '../../assets/imgMainPage.jpg';
+import GreenBlobImage from '../../assets/blob_green.png';
+import YellowBlobImage from '../../assets/blob_yellow.png';
+import PurpleBlobImage from '../../assets/blob_dusty_purple.png';
+import items from '../../data/data';
+import ArrowUp from '../../assets/icons/icons-arrow-up.png';
+import ArrowSearch from '../../assets/icons/icon-search.svg';
 
 const scrollToTop = () => {
   window.scrollTo({
     top: 0,
-    behavior: "smooth",
+    behavior: 'smooth',
   });
 };
-
 const HomePage = () => {
   const navigate = useNavigate();
-
   const search = () => {
-    navigate("/search");
+    navigate('/search');
   };
-
   return (
     <div className="flex flex-col w-10/12 my-auto lg:mx-auto mt-4 md:mt-12 my-12">
       <section className="flex flex-col lg:flex-row lg:justify-between xl:justify-between justify-center items-center">
-        <div className="lg:text-left text-center lg:w-2/5 items-center justify-center xl:justify-between">
+        <div className="lg:text-left  text-center lg:w-2/5 items-center justify-center xl:justify-between">
```

Comment:
The text in the blobs seems to be lining up better now! Quick tip if you want to have further customization of how the words stack or don't stack, you can wrap each word or phrase in a `span` allowing flex properties to be used on them. No need to do this, just a tip in case you weren't aware.